lib/action_cable/channel/base.rb

```diff
@@ -68,6 +68,8 @@ class Base
       include Callbacks
       include PeriodicTimers
       include Streams
+      include Naming
+      extend Broadcasting
```

Comment:
The `Broadcasting` methods are at the class scope because we don't have any instance to the channel when broadcasting, and we want to be able to generate the broadcasting name based on the channels name.

Doers that make sense?